Documentation/doc/Documentation/Tutorials/Tutorial_hello_world.txt

```diff
@@ -265,7 +265,7 @@ given by a direction, which is hardwired in the class
 
 \section intro_concept Concepts and Models
 
-In the previous section, we wrote that "Any model of the CGAL concept
+In the previous section, we wrote that: Any model of the CGAL concept
 Kernel provides what is required by the concept `ConvexHullTraits_2`.
```

Comment:
```suggestion
In the previous section, we wrote: any \em model of the CGAL \em concept
`Kernel` provides what is required by the concept `ConvexHullTraits_2`.
```

Reply:
I have chosen for a slightly different formulation, making both sentences 100% identical.

I also added the `\em` for the `model` and `concept` but I've seen that we have here a bigger inconsistency problem in the form that not everywhere where `model` or `concept` is used this is used the `\em` is used. Probably we should walk through everything though this is a very very big task, I just did a quick estimate of the number  of times the word model / concept appeared without the `\em`:
- model: 6311
- models: 925
- concept: 3938
- concepts: 1019

not all of these might refer to the CGAL model / concept word and need a `\em` (as model and concept are also regular words), but still ...

Reply:
I had put `\em` here simply because definitions were following up shortly; it was not intended to be something done systematically.

Reply:
@MaelRL, is this discussion solved?